third-party/phoneme/Makefile

```diff
@@ -3,7 +3,6 @@ PKG_NAME := phoneme
 PKG_VER  := trunk
```

Comment:
May be name `phoneme-trunk` will be more meaningful?

Reply:
Hm, I see here

```
PKG_NAME := phoneme
PKG_VER := trunk
```

Do you propose just PKG_NAME := phoneme-trunk?

Reply:
nvm, my mistake